### PR TITLE
Remove Looker Prow jobs

### PR DESCRIPTION
**Pull request by 2nd-half**
We are off GitHub, this is to avoid trigger Prow runs since repo is being synced.

### CODE REVIEW

**Review by chaodaiG — COMMENTED** [review]
/hold

there is a bug in prow https://github.com/kubernetes/test-infra/issues/25337 that prevents prow jobs from being GC'ed when the build cluster is deleted, please make sure deleting all prow jobs before deleting this build cluster:
- delete https://github.com/GoogleCloudPlatform/oss-test-infra/blob/master/prow/prowjobs/private-inrepoconfig-configcheck/looker/helltool/hellotool.yaml first
- uninstall prow app

**Comment by 2nd-half** [comment]
We have removed .prow configuration so no job would be running for a couple
of months

File: prow/oss/config.yaml [hunk 1]
```diff
@@ -83,13 +82,11 @@ tide:
 plank:
   job_url_prefix_config:
     '*': https://oss-prow.knative.dev/view/
-    'looker': https://oss-prow-private.knative.dev/view/
     "GoogleCloudPlatform/blueprints": https://prow.infra.cft.dev/view/
   pod_pending_timeout: 30m
   pod_unscheduled_timeout: 10m
   build_cluster_status_file: 'gs://oss-prow/oss-prow-build-clusters-statuses.json'
   report_templates:
```
Comment (chaodaiG) [inline prow/oss/config.yaml:89]: delete this as well

File: prow/oss/config.yaml [hunk 2]
```diff
@@ -260,28 +205,24 @@ deck:
     - compute-image-tools-test
     - gcp-guest-testgrid
   hidden_repos:
```
Comment (chaodaiG) [inline prow/oss/config.yaml:207]: delete this as well

File: prow/oss/config.yaml [hunk 3]
```diff
@@ -260,28 +205,24 @@ deck:
...
     "chaotoppicks": ["default"] # For experimenting purpose, will remove once done with experiments
     "GoogleCloudPlatform/blueprints": ["build-blueprints"]
     "GoogleCloudPlatform/oss-test-infra": ["default"]
-    "looker/helltool": ["build-looker-private", "build-looker-int-test", "build-looker-16", "build-looker-32"]
 
 # settings for github reporter from crier
 github_reporter:
   # summary_comment_repos is a list of orgs and org/repos for which failure report
   # comments is only sent when all jobs from current PR are finished.
   summary_comment_repos:
```
Comment (chaodaiG) [inline prow/oss/config.yaml:225]: delete this entirely

File: prow/oss/config.yaml [hunk 4]
```diff
@@ -309,10 +250,5 @@ presets:
...
 pubsub_triggers:
   - allowed_clusters:
-      - "build-looker-private"
-      - "build-looker-int-test"
-      - "build-looker-16"
-      - "build-looker-32"
-    project: 'prow-build-looker-github'
+    project:
     topics:
-      - "prow-test"
```
Comment (chaodaiG) [inline prow/oss/config.yaml:318]: delete this all

File: prow/oss/cluster/sinker.yaml [hunk 5]
```diff
@@ -27,32 +27,20 @@ spec:
         env:
         # Use KUBECONFIG envvar rather than --kubeconfig flag in order to provide multiple configs to merge.
         - name: KUBECONFIG
-          value: "/etc/kubeconfig/config-20211108:/etc/build-bms-oracle-team/kubeconfig:/etc/build-looker-32/kubeconfig:/etc/build-looker-16/kubeconfig:/etc/build-looker-int-test/kubeconfig:/etc/build-blueprints/kubeconfig:/etc/build-elcarro/kubeconfig:/etc/build-looker-private/kubeconfig:/etc/build-kubeflow/kubeconfig"
+          value: "/etc/kubeconfig/config-20211108:/etc/build-bms-oracle-team/kubeconfig:/etc/build-blueprints/kubeconfig:/etc/build-elcarro/kubeconfig:etc/build-kubeflow/kubeconfig"
```
Comment (chaodaiG) [inline prow/oss/cluster/sinker.yaml:30]:
```suggestion
          value: "/etc/kubeconfig/config-20211108:/etc/build-bms-oracle-team/kubeconfig:/etc/build-blueprints/kubeconfig:/etc/build-elcarro/kubeconfig:/etc/build-kubeflow/kubeconfig"
```

File: prow/oss/cluster/deck_private_deployment.yaml [hunk 6]
```diff
@@ -47,26 +47,14 @@ spec:
         env:
         # Use KUBECONFIG envvar rather than --kubeconfig flag in order to provide multiple configs to merge.
         - name: KUBECONFIG
-          value: "/etc/kubeconfig/config-20211108:/etc/build-bms-oracle-team/kubeconfig:/etc/build-looker-32/kubeconfig:/etc/build-looker-16/kubeconfig:/etc/build-looker-int-test/kubeconfig:/etc/build-elcarro/kubeconfig:/etc/build-looker-private/kubeconfig:/etc/build-kubeflow/kubeconfig"
+          value: "/etc/kubeconfig/config-20211108:/etc/build-bms-oracle-team/kubeconfig:/etc/build-elcarro/kubeconfig:/etc/build-kubeflow/kubeconfig"
```
Comment (chaodaiG) [inline prow/oss/cluster/deck_private_deployment.yaml:50]:
```suggestion
          value: "/etc/kubeconfig/config-20211108:/etc/build-bms-oracle-team/kubeconfig:/etc/build-blueprints/kubeconfig:/etc/build-elcarro/kubeconfig:/etc/build-kubeflow/kubeconfig"
```

File: prow/oss/config.yaml (hunk 3 above)
Comment (2nd-half) [reply prow/oss/config.yaml]: done

File: prow/oss/config.yaml (hunk 4 above)
Comment (2nd-half) [reply prow/oss/config.yaml]: done

**Review by chaodaiG — APPROVED** [review]

**Review by chaodaiG — APPROVED** [review]

**Review by chaodaiG — APPROVED** [review]

**Comment by chaodaiG** [comment]
surprisingly this is still not merged, @2nd-half could you rebase?

**Comment by chaodaiG** [comment]
the config validator failed, the job needs to be removed first, https://github.com/GoogleCloudPlatform/oss-test-infra/pull/1644 should do

**Comment by chaodaiG** [comment]
@2nd-half , please resolve the merge conflict and this PR should be good to go

**Pull request merged**